Resources/views/CRUD/base_acl_macro.html.twig

```diff
@@ -11,7 +11,8 @@ file that was distributed with this source code.
 
 {% macro render_form(form, permissions, td_type, admin, admin_pool, object) %}
     <form class="form-horizontal"
-          action="{{ admin.generateUrl('acl', {'id': admin.id(object), 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}" {{ form_enctype(form) }}
+          action="{{ admin.generateUrl('acl', {'id': admin.id(object), 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}"
+          {% if form.vars.multipart %} enctype="multipart/form-data"{% endif %}
```

Comment:
"Recommended" does not mean it match all cases ;) The sonata's form is a bit specific ...